### PR TITLE
Add Custom Teacher Course Creation Component

### PR DESCRIPTION
**Pull request by quinnbudan**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1145,9 +1145,9 @@
       }
     },
     "@brightspace-ui/core": {
-      "version": "1.59.0",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui/core/-/core-1.59.0.tgz",
-      "integrity": "sha512-6Q5pZv0PdpbU5aBb+lzHPxJx5Mvg2TfDFBMxFsI6G0tUJ6NUZzUiQ3yxUHawwCsTgn3gvW3sxFE7UZEv9nkrxw==",
+      "version": "1.59.2",
```
Comment (quinnbudan) [inline package-lock.json:1148]: FYI @svanherk

File: package-lock.json [hunk 2]
```diff
@@ -4614,7 +4614,7 @@
       }
     },
     "d2l-cpd": {
-      "version": "github:Brightspace/continuous-professional-development#340b71765a19eec750244010a1d8b228e45826c4",
+      "version": "github:Brightspace/continuous-professional-development#50ce1d8e058c40fe48f92d3b99ac6dbf493cd4da",
```
Comment (quinnbudan) [inline package-lock.json:4617]: FYI @BraydonKains

File: package-lock.json [hunk 3]
```diff
@@ -1227,18 +1227,22 @@
       }
     },
     "@d2l/video": {
-      "version": "github:Brightspace/d2l-video#5d3cc4898ff7d6ab50b2983a23bc85d820a9a6de",
-      "from": "github:Brightspace/d2l-video#semver:^2",
+      "version": "github:Brightspace/d2l-video#c3a0518ec3dfb70c89f93e778a88c6ce7da18dfb",
```
Comment (quinnbudan) [inline package-lock.json:1230]: FYI @ianlaidlaw

**Comment by quinnbudan** [comment]
Not bumping `aws-sdk` because of https://github.com/aws/aws-sdk-js/issues/3352

**Comment by ajredniwja** [comment]
> Not bumping `aws-sdk` because of [aws/aws-sdk-js#3352](https://github.com/aws/aws-sdk-js/issues/3352)

The issue has been fixed, it is safe to bump the version now.

**Comment by ajredniwja** [comment]
> > Not bumping `aws-sdk` because of [aws/aws-sdk-js#3352](https://github.com/aws/aws-sdk-js/issues/3352)
> 
> The issue has been fixed, it is safe to bump the version now.

I take that back the intended fix may not have been correctly implemented.

**Comment by ajredniwja** [comment]
> > > Not bumping `aws-sdk` because of [aws/aws-sdk-js#3352](https://github.com/aws/aws-sdk-js/issues/3352)
> > 
> > 
> > The issue has been fixed, it is safe to bump the version now.
> 
> I take that back the intended fix may not have been correctly implemented.

Apologies for the confusion the change is live now.

**Comment by quinnbudan** [comment]
> > > > Not bumping `aws-sdk` because of [aws/aws-sdk-js#3352](https://github.com/aws/aws-sdk-js/issues/3352)
> > > 
> > > 
> > > The issue has been fixed, it is safe to bump the version now.
> > 
> > 
> > I take that back the intended fix may not have been correctly implemented.
> 
> Apologies for the confusion the change is live now.

Thank you for the update! The version was bumped in another PR.

**Pull request merged**